### DISCUSSION TITLE
Pagination of <resource> list APIs

### DISCUSSION THREAD

**Discussion by jcasique** [juspay/hyperswitch#3735] [Ideas / Feature Requests]
It would be nice to add pagination to all resource list endpoints, since we need to handle a large list of them.

**Comment by Narayanbhat166** [comment]
Hey @jcasique. Yes, you are right. There are few good first issues related to this. You can track their progress here.
- https://github.com/juspay/hyperswitch/issues/3746.
- https://github.com/juspay/hyperswitch/issues/3748
- https://github.com/juspay/hyperswitch/issues/3749
